apps/super-random-ts/generate.sh

```diff
@@ -0,0 +1,19 @@
+set -o errexit
+set -o pipefail
+
+PULUMI_BUILD_CONTAINER_VERSION=v0.3.0
+
+# First build the image for the Pulumi Build Container
+echo "* Generating go-components SDK:"
+pulumi package add ../../go
+
+echo "* Generating node-components SDK:"
+pulumi package add ../../ts
+
+echo "* Generating py-components SDK:"
+pulumi package add ../../py
+
+echo "* Generating dotnet-components SDK:"
+pulumi package add ../../dotnet
```

Comment:
You shouldn't need this file - just run `pulumi install`

Reply:
Really? pulumi install does code generation?  Will give it a try, thanks.

Reply:
Sorry.I haven't had an opportunity to try this.  @mikhailshilkov to be clear, did you say that doing `pulumi install` on the  super-random example would automatically re-generate the SDKs?  Or, what are you suggesting?  Thanks.

Reply:
Yes

Reply:
So, do you want to remove this shell script and replace it with `packages`?